### PR TITLE
Transações - Fixa valor de timezone para America/Sao_Paulo

### PR DESCRIPTION
**Pull request by vagnervst**
## Contexto
A função `moment.tz.guess` pode retornar valores inesperados incompatíveis com o Elasticsearch, este PR ajusta este valor a fim de tentar mitigar os erros que estão ocorrendo na tela de transações.

## Checklist
- [x] Atualiza timezone de transactions para `America/Sao_Paulo`

## Como testar
1. Acesse a tela de transações e verifique se a lista está carregando

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7813f833b882b9a9b179c9b8aeebf5b70b4881dd
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7813f833b882b9a9b179c9b8aeebf5b70b4881dd
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7813f833b882b9a9b179c9b8aeebf5b70b4881dd
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7813f833b882b9a9b179c9b8aeebf5b70b4881dd
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7813f833b882b9a9b179c9b8aeebf5b70b4881dd
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 345f8e534d8dbe7ef6b17d835d53c5f5e98901c0
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 345f8e534d8dbe7ef6b17d835d53c5f5e98901c0
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 345f8e534d8dbe7ef6b17d835d53c5f5e98901c0
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a2980d98d3fbc5cb21d692a86c6f3c70bd375b16
  *   
￼ Good titles doesn't exceed 50 characters. To add more details use the body!

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by marisamodolo — APPROVED** [review]

**Review by marisamodolo — APPROVED** [review]

File: packages/cockpit/src/transactions/search/query.js [hunk 1]
```diff
@@ -102,7 +101,7 @@ const aggregations = {
       field: 'date_created',
       interval: 'day',
       format: 'MM/dd/yyyy',
-      time_zone: moment.tz.guess(),
+      time_zone: 'America/Sao_Paulo',
```
Comment (itsdaiego) [inline packages/cockpit/src/transactions/search/query.js:104]: PLEASE LORD MAY YOU ALLOW THIS TO WORK :pray:

**Review by itsdaiego — APPROVED** [review]

**Review by Hrfreire — APPROVED** [review]

**Pull request merged**